securesystemslib/key.py

```diff
@@ -63,16 +60,16 @@ class SSlibKey(Key):
 
     def __init__(
         self,
-        keyid: Optional[str],
         keytype: str,
         scheme: str,
         keyval: Dict[str, str],
+        keyid: Optional[str] = None,
```

Comment:
Can you move all changes from this file as part of the commit https://github.com/in-toto/securesystemslib/pull/2/commits/5c18b8905bd419057c50d6c2b9673ebb0db8fb1c as it doesn't make sense for those changes to be part of this commit which focuses on testing.

Reply:
Done